package.json

```diff
@@ -16,6 +16,7 @@
   "scripts": {
     "build": "rollup -c",
     "lint": "eslint --ext .js src",
+    "cleanCss": "node clean-scss.js",
```

Comment:
I'm not keen on camel-case for script names. `clean`, `clean:css` or `clean-css` instead?

Reply:
Done